Dev (#47)
42-AI/bootcamp_data-engineering: day01/day01.md

# Bootcamp Data Engineering

# Day01 - NoSQL with Elasticsearch

Today, you will learn how to use a NoSQL database: Elasticsearch.

> Wait... Elasticsearch is a database? Well, not exactly it is more than that. It is defined as a search and analytics engine. But let's keep it simple for now, consider it as a database, we will see the rest later.

Reviewer: ```suggestion
Wait... Elasticsearch is a database? Well, not exactly. It is more than that. It is defined as a search and analytics engine. But let's keep it simple for now. Consider it as a database and we will see the rest later.
```

## Notions of the day

> Create an Elasticsearch cluster, create index and mappings, ingest document, search & aggregate, create visuals with Kibana.  

Reviewer: ```suggestion
Create an Elasticsearch cluster, create index and mappings, ingest documents, search & aggregate, produce visuals with Kibana.  
```

> In the first part of this day (ex00 to ex03) you will learn the basics of Elasticsearch. Then you will apply this to a real dataset.

Reviewer: ```suggestion
In the first part of this day (ex00 to ex03) you will learn the basics of Elasticsearch. Then you will apply your newly learned skills to a real dataset.
```
